app/test/integration-test/src/test/java/io/syndesis/test/itest/extensions/IntegrationDependencies_IT.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.syndesis.test.itest.extensions;
+
+import java.net.URISyntaxException;
+
+import io.syndesis.common.model.Dependency;
+import io.syndesis.common.model.action.ConnectorAction;
+import io.syndesis.common.model.action.ConnectorDescriptor;
+import io.syndesis.common.model.connection.ConfigurationProperty;
+import io.syndesis.common.model.connection.Connection;
+import io.syndesis.common.model.connection.Connector;
+import io.syndesis.common.model.integration.Flow;
+import io.syndesis.common.model.integration.Integration;
+import io.syndesis.common.model.integration.Step;
+import io.syndesis.common.model.integration.StepKind;
+import io.syndesis.test.container.integration.SyndesisIntegrationRuntimeContainer;
+import io.syndesis.test.itest.SyndesisIntegrationTestSupport;
+import org.junit.Test;
+import org.springframework.test.context.ContextConfiguration;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+@ContextConfiguration(classes = IntegrationDependencies_IT.EndpointConfig.class)
+public class IntegrationDependencies_IT extends SyndesisIntegrationTestSupport {
+
+    @Test
+    public void shouldVerifyDependecyClasspath() throws URISyntaxException {
+        // We need an S2I to load extensions in classpath
+        System.setProperty("syndesis.s2i.build.enabled", "true");
+        System.setProperty("syndesis.debug.enabled", "false");
```

Comment:
This flag could be set on the `SyndesisIntegrationRuntimeContainer.Builder`. There is already a possibility to set `enableDebug()`

```
new SyndesisIntegrationRuntimeContainer.Builder()
            .name("bean-to-log")
            .enableDebug()
```

So we need to refactor/add something to disable debug on the builder.